packages/core/src/provisioning.ts

```diff
@@ -1,10 +1,17 @@
+import { createServer, IncomingMessage, ServerResponse } from "http";
+import { Config, ConfigParam } from "./config";
 import { Account, AccountID } from "./account";
 import { AsSerializable, Serializable } from "./encoding";
 import { Err, ErrorCode } from "./error";
 import { Org, OrgID, OrgInfo } from "./org";
 import { Storable, Storage } from "./storage";
 import { getIdFromEmail } from "./util";
 
+export class BasicProvisionerConfig extends Config {
```

Comment:
@MaKleSoft I was going to implement SCIM in `BasicProvisioning`, but I'm running into some problems like:

1. `core` needs to run on browser (I "mocked" `http` via `webpack`)
2. The methods I find for inviting/adding members require the app to be unlocked, which can't be done via server like this

I initially looked into hooking up to the `api.ts` provisioning in `server`, but it seems like that's not used? (only `basic` and `stripe` are allowed) I could add this to the `stripe` provider, though, and just allow only some provisioners to support it, which would make sense.

Reply:
As discussed, this doesn't need to live in core. Let's just make the whole thing it's own class in `@padloc/server` for now.